main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsFareAttributeTransfersEnum.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsEnumValue;
+
+@GtfsEnumValue(name = "NO_TRANSFER", value = 0)
```

Comment:
This field is quite special in the spec ;) It is a "required that can be empty" and its "empty" value differs from all its numeric values. We may introduce the magical -1, as you suggested, or we can simply use the `hasTransfers()` method to tell if this field is empty. Both ways do not look elegant. Perhaps a magical -1 is less error prone but using `hasTransfers()` is less hackish. Which one do you prefer?

Reply:
>This field is quite special in the spec ;)

Yes, I thought the same thing when I looked at it. It's the only field defined this way in GTFS from what I saw, which makes it very much a corner case. I'd err on the side of being less error prone in this case, so I'd prefer the magical -1 solution.

Reply:
OK, I added this magical -1. However, this allows to specify that -1 explicitly in the CSV file - which would be against the GTFS spec. Are you OK with that?

Reply:
>However, this allows to specify that -1 explicitly in the CSV file - which would be against the GTFS spec. Are you OK with that?

Ugh - no, we shouldn't allow that. We need to only support canonical GTFS values. Can we still flag -1 as invalid somehow? If it's not possible to only set `UNLIMITED` in the case where the field is empty (and still flag a `-1` value in the GTFS data as an error), then I believe our only choice as you mentioned previously is to rely on calling code to use `hasTransfers()`. In that case, we should add as much documentation as possible in the Javadocs to explain this behavior.

Reply:
I restored the previous situation. This is the simplest way to achieve what we need: this field may be either empty or keep 0, 1 or 2, and this is validated automatically.

I can tell you that Google does not implement any specific checks based on concrete value of this field, so the risk that someone forgets to call `hasTransfers()` is very low.

Anyway, I would advocate for adding -1 (or another explicit value) to the GTFS standard and also marking this field as `Optional` instead of `Required` in the transit/reference.md.

Reply:
@aababilov Sounds good, thanks! I'll pass on the comments re: spec changes to MobilityData spec team.

Reply:
Unfortunately, this change would not be backward compatible, so impossible to pass under the current GTFS amendment process. However, this could move forward if GTFS Schedule was versioned. I added the issue with this field in our records!